docs/context-pipeline-integration.md

```
@@ -0,0 +1,231 @@
+if (contextText && contextText.trim().length > 0) {
+  const contextMessage: Content = {
+    role: 'user',
+    parts: [{ text: `[SYSTEM CONTEXT]\n${contextText}\n[/SYSTEM CONTEXT]` }]
+  };
+  requestContents = this.getHistory(true).concat([contextMessage, userContent]);
+}
+```
+
+## Features
+
+### 1. Automatic Context Retrieval
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)